GUI.py

```diff
@@ -153,11 +196,16 @@ def plot(self):
                 self.messege(
                     "无法绘制！\nX与Y的维度不同" if self.language == "zh_CN" else "Cannot plot!\nX and Y have different dimensions")
             else:
-                self.graphWidget.clear()
-                scatter = pg.ScatterPlotItem(pen=pg.mkPen(width=1, color='k'), symbol='o', size=4)
-                self.graphWidget.addItem(scatter)
-                pos = [{'pos': [x[i], y[i]]} for i in range(len(x))]
-                scatter.setData(pos)
+
+                self.full_x = x
+                self.full_y = y
+
+                self.sc.axes.clear()
+                self.sc.axes.scatter(x, y, s=5, c='k', label='Raw data', picker=True, pickradius=1)
+                self.outliers = np.zeros(x.shape).astype(bool)
+                self.sc.mpl_connect('pick_event', self.outlier_pick)

```

Comment:
Used to connect the plot to an update refresh that will select the point in close proximity and re-run the fit.